cocos/base/CCData.cpp

```diff
@@ -126,7 +134,7 @@ void Data::fastSet(unsigned char* bytes, const ssize_t size)
 
 void Data::clear()
 {
-    free(_bytes);
+    if(!_bytes) free(_bytes);
```

Comment:
Here is wrong check.
Should be if(_bytes)

Reply:
my fault